locales/en-US/experiments.ftl

```diff
@@ -29,6 +29,20 @@ cliqzToursteps0Copy = Start typing in the URL bar to see real-time results.
 cliqzToursteps1Copy = Open a new tab and add your favorite websites for one-click access.
 cliqzToursteps2Copy = You can always give us feedback or disable Cliqz from Test Pilot.
 cliqzEolwarning = This experiment is ending soon. If you have it enabled, you can keep it for now, and we will help you migrate to the next version when it becomes available.
+colorDescription = Show your colors. Create and share your own Firefox themes with just a few clicks.
+colorIntroduction = Firefox Color makes it easy for anyone to change the look of Firefox. It’s a simple tool that lets you choose different color combinations and background textures for your own Firefox theme.
```

Comment:
Can `Firefox Color` be localized or not? Please add a note explaining what's the case

Reply:
I can add a note to the ftl, but it will get blown away every time the yaml gets recompiled. I'm not sure there's any way to persist such a note across PRs for experiment details.

Reply:
Just for GP -- what was the answer to being localized or not?

Reply:
Yeah, there's no way to leave comments in this FTL because they'll get blown away by the build process. We should merge this as is. I've filed #3563 to handle this in future